WARNING 2021-08-09 21:13:18,057 log 224 Not Found: /register/login.html
WARNING 2021-08-09 21:13:18,058 basehttp 161 "GET /register/login.html HTTP/1.1" 404 2305
WARNING 2021-08-09 21:13:27,830 log 224 Not Found: /register/login.html
WARNING 2021-08-09 21:13:27,831 basehttp 161 "GET /register/login.html HTTP/1.1" 404 2305
INFO 2021-08-09 21:13:32,423 basehttp 161 "GET /register HTTP/1.1" 301 0
INFO 2021-08-09 21:13:32,432 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
INFO 2021-08-09 21:14:03,446 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
INFO 2021-08-09 21:18:04,106 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
INFO 2021-08-09 21:20:31,243 autoreload 251 C:\Users\suze\Desktop\kira\users\urls.py changed, reloading.
INFO 2021-08-09 21:20:32,352 autoreload 637 Watching for file changes with StatReloader
WARNING 2021-08-09 21:23:11,936 log 224 Not Found: /
WARNING 2021-08-09 21:23:11,938 basehttp 161 "GET / HTTP/1.1" 404 2403
INFO 2021-08-09 21:23:50,596 autoreload 251 C:\Users\suze\Desktop\kira\users\urls.py changed, reloading.
INFO 2021-08-09 21:23:51,382 autoreload 637 Watching for file changes with StatReloader
INFO 2021-08-09 21:23:52,321 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
INFO 2021-08-09 21:25:41,640 autoreload 251 C:\Users\suze\Desktop\kira\users\views.py changed, reloading.
INFO 2021-08-09 21:25:42,845 autoreload 637 Watching for file changes with StatReloader
INFO 2021-08-09 21:25:43,102 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
WARNING 2021-08-09 21:25:46,033 log 224 Not Found: /register/login.html
WARNING 2021-08-09 21:25:46,035 basehttp 161 "GET /register/login.html HTTP/1.1" 404 2472
WARNING 2021-08-09 21:26:30,561 log 224 Not Found: /
WARNING 2021-08-09 21:26:30,562 basehttp 161 "GET / HTTP/1.1" 404 2397
INFO 2021-08-09 21:26:36,616 basehttp 161 "GET /login HTTP/1.1" 301 0
ERROR 2021-08-09 21:26:36,719 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 21:26:36,731 basehttp 161 "GET /login/ HTTP/1.1" 500 145906
WARNING 2021-08-09 21:27:31,712 log 224 Not Found: /re
WARNING 2021-08-09 21:27:31,714 basehttp 161 "GET /re HTTP/1.1" 404 2421
INFO 2021-08-09 21:27:36,950 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
ERROR 2021-08-09 21:27:52,160 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 21:27:52,167 basehttp 161 "GET /login/ HTTP/1.1" 500 145906
WARNING 2021-08-09 21:27:58,482 log 224 Not Found: /login.html
WARNING 2021-08-09 21:27:58,484 basehttp 161 "GET /login.html HTTP/1.1" 404 2445
WARNING 2021-08-09 21:28:01,423 log 224 Not Found: /login.html
WARNING 2021-08-09 21:28:01,424 basehttp 161 "GET /login.html HTTP/1.1" 404 2445
INFO 2021-08-09 21:28:18,033 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
ERROR 2021-08-09 21:28:26,956 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 21:28:26,976 basehttp 161 "GET /login/ HTTP/1.1" 500 145906
INFO 2021-08-09 21:29:05,871 autoreload 637 Watching for file changes with StatReloader
ERROR 2021-08-09 21:29:14,021 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 21:29:14,030 basehttp 161 "GET /login/ HTTP/1.1" 500 146043
INFO 2021-08-09 21:30:21,393 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
WARNING 2021-08-09 21:30:25,522 log 224 Not Found: /lodin
WARNING 2021-08-09 21:30:25,522 basehttp 161 "GET /lodin HTTP/1.1" 404 2430
ERROR 2021-08-09 21:30:30,459 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 21:30:30,466 basehttp 161 "GET /login/ HTTP/1.1" 500 145906
INFO 2021-08-09 22:37:18,024 autoreload 637 Watching for file changes with StatReloader
INFO 2021-08-09 22:44:21,169 autoreload 637 Watching for file changes with StatReloader
INFO 2021-08-09 22:44:45,259 autoreload 637 Watching for file changes with StatReloader
WARNING 2021-08-09 22:44:46,798 log 224 Not Found: /
WARNING 2021-08-09 22:44:46,799 basehttp 161 "GET / HTTP/1.1" 404 2397
WARNING 2021-08-09 22:44:46,950 log 224 Not Found: /favicon.ico
WARNING 2021-08-09 22:44:46,950 basehttp 161 "GET /favicon.ico HTTP/1.1" 404 2448
INFO 2021-08-09 22:44:51,570 basehttp 161 "GET /admin HTTP/1.1" 301 0
INFO 2021-08-09 22:44:51,587 basehttp 161 "GET /admin/ HTTP/1.1" 302 0
INFO 2021-08-09 22:44:51,619 basehttp 161 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 2214
INFO 2021-08-09 22:44:51,650 basehttp 161 "GET /static/admin/css/base.css HTTP/1.1" 200 19513
INFO 2021-08-09 22:44:51,662 basehttp 161 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2271
INFO 2021-08-09 22:44:51,662 basehttp 161 "GET /static/admin/css/login.css HTTP/1.1" 200 939
INFO 2021-08-09 22:44:51,664 basehttp 161 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 1360
INFO 2021-08-09 22:44:51,665 basehttp 161 "GET /static/admin/css/responsive.css HTTP/1.1" 200 18545
INFO 2021-08-09 22:44:51,677 basehttp 161 "GET /static/admin/css/fonts.css HTTP/1.1" 200 423
INFO 2021-08-09 22:44:51,748 basehttp 161 "GET /static/admin/fonts/Roboto-Light-webfont.woff HTTP/1.1" 200 85692
INFO 2021-08-09 22:44:51,749 basehttp 161 "GET /static/admin/fonts/Roboto-Regular-webfont.woff HTTP/1.1" 200 85876
INFO 2021-08-09 22:44:56,719 basehttp 161 "GET /register/ HTTP/1.1" 200 5188
ERROR 2021-08-09 22:45:01,202 log 224 Internal Server Error: /login/
Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 470, in parse
    compile_func = self.tags[command]
KeyError: 'static'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\exception.py", line 47, in inner
    response = get_response(request)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\core\handlers\base.py", line 181, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 70, in view
    return self.dispatch(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\views\generic\base.py", line 98, in dispatch
    return handler(request, *args, **kwargs)
  File "C:\Users\suze\Desktop\kira\users\views.py", line 15, in get
    return render(request, 'login.html')
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\shortcuts.py", line 19, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 143, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\engine.py", line 125, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\loaders\base.py", line 29, in get_template
    return Template(
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 155, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 193, in compile_nodelist
    return parser.parse()
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 472, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\suze\Desktop\kira\venv\lib\site-packages\django\template\base.py", line 531, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 11: 'static'. Did you forget to register or load this tag?
ERROR 2021-08-09 22:45:01,207 basehttp 161 "GET /login/ HTTP/1.1" 500 146637
INFO 2021-08-09 22:45:53,813 basehttp 161 "GET /login/ HTTP/1.1" 200 3068
INFO 2021-08-09 22:45:53,852 basehttp 161 "GET /static/js/login.js HTTP/1.1" 200 677
WARNING 2021-08-09 22:46:00,557 log 224 Not Found: /login/register.html
WARNING 2021-08-09 22:46:00,558 basehttp 161 "GET /login/register.html HTTP/1.1" 404 2472
INFO 2021-08-09 22:46:47,832 basehttp 161 "GET /login/ HTTP/1.1" 200 3068
INFO 2021-08-09 22:46:50,594 basehttp 161 "GET /login/ HTTP/1.1" 200 3068
WARNING 2021-08-09 22:46:52,591 log 224 Not Found: /login/register.html
WARNING 2021-08-09 22:46:52,592 basehttp 161 "GET /login/register.html HTTP/1.1" 404 2472
INFO 2021-08-09 22:46:56,887 basehttp 161 "GET /register/ HTTP/1.1" 200 5185
INFO 2021-08-09 22:46:58,345 basehttp 161 "GET /login/ HTTP/1.1" 200 3068
INFO 2021-08-09 22:47:22,252 basehttp 161 "GET /login/ HTTP/1.1" 200 3064
WARNING 2021-08-09 22:47:23,800 log 224 Not Found: /login/register/
WARNING 2021-08-09 22:47:23,801 basehttp 161 "GET /login/register/ HTTP/1.1" 404 2460
WARNING 2021-08-09 22:47:31,343 log 224 Not Found: /login/register/
WARNING 2021-08-09 22:47:31,343 basehttp 161 "GET /login/register/ HTTP/1.1" 404 2460
WARNING 2021-08-09 22:47:32,451 log 224 Not Found: /login/register/
WARNING 2021-08-09 22:47:32,452 basehttp 161 "GET /login/register/ HTTP/1.1" 404 2460
INFO 2021-08-09 22:47:34,624 basehttp 161 "GET /login/ HTTP/1.1" 200 3065
INFO 2021-08-09 22:47:35,666 basehttp 161 "GET /register/ HTTP/1.1" 200 5185
INFO 2021-08-09 22:47:37,642 basehttp 161 "GET /login/ HTTP/1.1" 200 3065
INFO 2021-08-10 01:17:13,408 autoreload 637 Watching for file changes with StatReloader
WARNING 2021-08-10 01:17:17,835 log 224 Not Found: /
WARNING 2021-08-10 01:17:17,837 basehttp 161 "GET / HTTP/1.1" 404 2397
INFO 2021-08-10 01:17:22,375 basehttp 161 "GET /register/ HTTP/1.1" 200 5185
INFO 2021-08-10 01:17:23,893 basehttp 161 "GET /login/ HTTP/1.1" 200 3065
INFO 2021-08-10 01:17:26,282 basehttp 161 "GET /register/ HTTP/1.1" 200 5185
INFO 2021-08-10 01:17:28,224 basehttp 161 "GET /login/ HTTP/1.1" 200 3065
INFO 2021-08-10 01:17:33,065 basehttp 161 "GET /register/ HTTP/1.1" 200 5185
INFO 2021-08-10 01:26:24,170 autoreload 251 C:\Users\suze\Desktop\kira\users\models.py changed, reloading.
INFO 2021-08-10 01:26:24,900 autoreload 637 Watching for file changes with StatReloader
INFO 2021-08-10 01:26:56,143 autoreload 251 C:\Users\suze\Desktop\kira\users\models.py changed, reloading.
